src/style.css

```
@@ -1 +1,84 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  margin: auto 0;
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+.logo {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.logo__image {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.nav__item {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.nav__link {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  font-weight: 500px;
```

`font-weight` is used without any dimension, just value
```suggestion
  font-weight: 500;
```